deployment/terraform/create.go

```diff
@@ -338,6 +344,16 @@ func (t *Terraform) updateAppConfig(ip string, sshc *ssh.Client, output *Output)
 	cfg.PluginSettings.Enable = model.NewBool(true)
 	cfg.PluginSettings.EnableUploads = model.NewBool(true)
 
+	if t.config.HasElasticSearch() {
```

Comment:
We usually check the output struct for this. It helps confirm that Value[0] is actually there. We can add a similar method to the output struct also like `HasProxy`.